lib/validation.js

```diff
@@ -122,10 +122,24 @@ function wrapValidationError (result, dataVar) {
   return error
 }
 
+// no idea about performance
+function _isJsonString (data) {
+  try {
+    JSON.parse(data)
+    return true
+  } catch (e) {
+    return false
+  }
+}
+
 function serialize (context, data, statusCode) {
   var responseSchemaDef = context[responseSchema]
   if (!responseSchemaDef) {
-    return JSON.stringify(data)
+    if (typeof data === 'string' && _isJsonString(data)) {
```

Comment:
I think that it can affect the performance.... if we receive a parameter to explicit it?

Reply:
Yes, it'll be more efficient solution, but i don't know how to pass param here...

Reply:
I'll try to pass it to context